templates/dashboard/contact_detail.html

```diff
@@ -70,4 +72,37 @@ <h1 class="h3">
   </div>
 {% endblock %}
 
+{% block script %}
+    {% if current_user.is_premium() %}
+    <script>
+    const MAX_BYTES = 2048; // 2KiB
+
+    function drop(event) {
+        event.stopPropagation();
+        event.preventDefault();
+
+        let files = event.dataTransfer.files;
+        for (let i = 0; i < files.length; i++) {
+            let file = files[i];
+            console.log(file, file.name);
+            if(file.type !== 'text/plain'){
+                toastr.warning(`File ${file.name} is not a public key file`);
+                continue;
+            }
+            let reader = new FileReader();
+            reader.onloadend = onFileLoaded;
+            reader.readAsBinaryString(file);
+        }
+    }
 
+    function onFileLoaded(event) {
+        const initialData = event.currentTarget.result.substr(0, MAX_BYTES);
+        $('#pgp-public-key').val(initialData);
+        console.log($('#pgp-public-key'));
```

Comment:
the `console.log` can be removed I guess :)

Reply:
Oops. Removed.